codespell_lib/data/dictionary.txt

```diff
@@ -702,6 +702,7 @@ bracese->braces
 Brasillian->Brazilian
 breakthough->breakthrough
 breakthroughts->breakthroughs
+breef->brief
```

Comment:
given the proximity of `e` to `r` on a standard keyboard I guess `beef` should also be offered

Reply:
standard US keyboard I mean

Reply:
point taken, revised commit